history/cultures/lordaeronian.txt

```diff
@@ -0,0 +1,49 @@
+1.1.1 = {
+	discover_innovation = innovation_barracks 
+	discover_innovation = innovation_mustering_grounds 
+	#
+	discover_innovation = innovation_city_planning 
+	discover_innovation = innovation_currency_01 
+	#
+}
+270.1.1 = {
+	discover_innovation = innovation_bannus 
+	discover_innovation = innovation_catapult 
+	#
+	discover_innovation = innovation_ledger 
+	discover_innovation = innovation_development_01 
+	discover_innovation = innovation_plenary_assemblies 
+	#
+}
+400.1.1 = {
+	discover_innovation = innovation_motte 
+	discover_innovation = innovation_quilted_armor 
+	#
+	discover_innovation = innovation_casus_belli 
+	discover_innovation = innovation_gavelkind 
+	discover_innovation = innovation_crop_rotation 
+	#
+	join_era = culture_era_early_medieval
+}
+508.1.1 = {
+	discover_innovation  = innovation_battlements 
+	#
+	discover_innovation = innovation_baliffs 
+	discover_innovation = innovation_hereditary_rule 
+	#
+}
+583.1.1 = {
+	discover_innovation  = innovation_horseshoes 
+	discover_innovation  = innovation_mangonel 
+	#
+	discover_innovation = innovation_manorialism 
+	discover_innovation = innovation_currency_02 
+}
+605.1.1 = {
+	discover_innovation  = innovation_burhs 
+	discover_innovation  = innovation_arched_saddle 
```

Comment:
- We know for a fact that armored horses existed on Warcraft II (https://wow.gamepedia.com/Knight_(Warcraft_II)). Eastern Kigndom human group (Lordaeronian, Azerothian, Alteraci, Hilsbradian, and Stromic) should have at least ``discover_innovation  = innovation_arched_saddle`` after 587.1.1 (https://wow.gamepedia.com/Timeline#Second_War)